src/components/app/app.tsx

```diff
@@ -50,10 +47,15 @@ const EventVisualizer: FC = () => {
             <GridCenterColumnBottom>
                 <div style={{ padding: 8 }}>
                     <h1>Visualization Canvas</h1>
-                    <h2>
-                        {i18n.t('Hello {{name}}', { name: currentUser.name })}
-                    </h2>
                     <h3>{i18n.t('Welcome to DHIS2 with TypeScript!')}</h3>
+                    {currentVis && (
```

Comment:
We discussed this part could be replaced with the typeguard. But we also discussed that perhaps a better type organisation could be relevant here. I guess we can tackle this part when we implement the "Getting started" / landing page work.